src/openvic-simulation/country/CountryInstance.hpp

```diff
@@ -139,7 +139,7 @@ namespace OpenVic {
 		// TODO - population change over last 30 days
 		fixed_point_t PROPERTY(national_consciousness);
 		fixed_point_t PROPERTY(national_militancy);
-		IndexedMap<PopType, fixed_point_t> PROPERTY(pop_type_distribution);
+		IndexedMap<PopType, Pop::pop_size_t> PROPERTY(pop_type_distribution);
```

Comment:
The reason for this using `fixed_point_t` was partly because pie charts require their data to be `fixed_point`s. We'll need a corresponding main repo PR to add support for non-`fixed_point_t` value types for pie chart data.